x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/GenerateInitialBuiltinUsersPasswordListener.java

```diff
@@ -36,6 +38,11 @@ public GenerateInitialBuiltinUsersPasswordListener(NativeUsersStore nativeUsersS
 
     @Override
     public void accept(SecurityIndexManager.State previousState, SecurityIndexManager.State currentState) {
+        final PrintStream out = BootstrapInfo.getOriginalStandardOut();
+        if (null == out) {
```

Comment:
Now is this case possible? We could add a not null assertion when setting in BootstrapInfo?

Reply:
Yes, you are right. There should be nothing else calling `BootstrapInfo#init` and there is no setter for `originalStandardOut` so I think we are ok ?